AjaxControlToolkit.SampleSite/Global.asax

```diff
@@ -11,6 +11,8 @@
             "~/Scripts/WebForms/MsAjax/MicrosoftAjaxTimer.js",
             "~/Scripts/WebForms/MsAjax/MicrosoftAjaxWebForms.js"));
 
+        // In Medium Trust environments, set AjaxFileUploadHelper.RootTempFolderPath to an existing directory located within the web application root.
+        AjaxFileUploadHelper.RootTempFolderPath = HttpContext.Current.Server.MapPath("~/Temp");
```

Comment:
No, we don't. We have to add a "placeholder" file to get this folder indexed by git.